src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -549,4 +557,32 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             controller.dispose();
         });
     }
+
+    // Load up a python kernel to just launch in the current vscode environment
+    private loadCurrentPythonEnvironmentKernel() {
+        const config = this.configuration.getSettings();
+
+        if (config.currentPythonEnvironmentKernel) {
```

Comment:
Can we put this behind an experiment?
And optionally enable this for all new users (VS Code providse a way to identify new users)